state/store.go

```diff
@@ -298,10 +298,12 @@ func (store dbStore) PruneStates(from int64, to int64) error {
 			}
 
 			if p.ConsensusParams.Equal(&tmproto.ConsensusParams{}) {
-				p.ConsensusParams, err = store.LoadConsensusParams(h)
+				var params types.ConsensusParams
+				params, err = store.LoadConsensusParams(h)
```

Comment:
```suggestion
				params, err := store.LoadConsensusParams(h)
```

Not sure the separated variable declaration gets us anything.